includes/ApplHandlingByStr.php

```diff
@@ -41,6 +82,44 @@ function showStatus( $stat_str )
 }
 
 
+function needAcceptor($status)
+{
+	$str = explode ( ';', $status );
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Space before opening parenthesis of function call prohibited
* Space after opening parenthesis of function call prohibited
* Expected 0 spaces before closing bracket; 1 found

<!-- Comment by [SideCI](https://sideci.com) -->